index.html

```diff
@@ -74,8 +74,13 @@ <h1 class="toblur">
                 </ul>
             </div>
         </section>
+
+        <section class="modal">
+
+        </section>
```

Comment:
- [ ] Nice implementing the pop up window, however the technologies and the button should be in the right side in the desktop version as the ones in the Figma design so please try to fix that in order to match Figma design 

- [ ] Figma design
![image](https://user-images.githubusercontent.com/61790587/167958435-044a56c3-529d-4466-bda8-1588c8e27f72.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/167958446-280123ef-92d6-4dff-9713-aabd3204d99f.png)